cpu/stm32_common/periph/i2c_2.c

```diff
@@ -16,9 +17,9 @@
  * @file
  * @brief       Low-level I2C driver implementation
  *
- * This driver supports the STM32 F4 families.
+ * This driver supports the STM32 F1, L1, and F4 families.
```

Comment:
and F2

Reply:
Yup thanks,  I will split everything out.